doc/modules/linear_model.rst

```diff
@@ -1239,6 +1239,6 @@ This way, we can solve the XOR problem with a linear classifier::
            [ 1.,  0.,  1.,  0.],
            [ 1.,  1.,  0.,  0.],
            [ 1.,  1.,  1.,  1.]])
-    >>> clf = Perceptron(fit_intercept=False, n_iter=10, shuffle=False).fit(X, y)
-    >>> clf.score(X, y)
+    >>> reg = Perceptron(fit_intercept=False, n_iter=10, shuffle=False).fit(X, y)
```

Comment:
Good catch!

--Hobson
(503) 974-6274
gh <https://github.com/hobson/> twtr <https://twitter.com/hobsonlane> li
<https://www.linkedin.com/in/hobsonlane> g+
<http://plus.google.com/+HobsonLane/> so
<http://stackoverflow.com/users/623735/hobs>

On Wed, Aug 31, 2016 at 3:16 PM, Andreas Mueller <notifications@github.com>
wrote:

> In doc/modules/linear_model.rst
> <https://github.com/scikit-learn/scikit-learn/pull/7316#discussion_r77083652>
> :
>
> > @@ -1239,6 +1239,6 @@ This way, we can solve the XOR problem with a linear classifier::
> >             [ 1.,  0.,  1.,  0.],
> >             [ 1.,  1.,  0.,  0.],
> >             [ 1.,  1.,  1.,  1.]])
> > -    >>> clf = Perceptron(fit_intercept=False, n_iter=10, shuffle=False).fit(X, y)
> > -    >>> clf.score(X, y)
> > +    >>> reg = Perceptron(fit_intercept=False, n_iter=10, shuffle=False).fit(X, y)
>
> this is a classifier ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/7316/files/9f659983769e9e15180e4f4ce8d81534ba979598#r77083652>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAR39xmxg6aF1JbT9ulCVm0Yb5bOUkFoks5qlf0jgaJpZM4JyGxM>
> .
>